libs/langgraph/langgraph/checkpoint/base.py

```diff
@@ -53,6 +54,10 @@ class CheckpointMetadata(TypedDict, total=False):
     """
 
 
+class TaskInfo(TypedDict):
+    status: Literal["scheduled", "success", "error"]
```

Comment:
it's more because scheduled the transition from scheduled to processing is handled by your queueing system, so we don't need to track here too. we're not using this yet (until we write the first queue adapter)